style.css

```diff
@@ -339,15 +328,331 @@ a {
+}
+
+.categories-red5 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/10/google-logo-podium.png);
+}
+
+.categories-white6 {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(152, 154, 163, 0.52),
+      rgb(7, 1, 5)
+    ),
+    url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/zoom-fundraising-gq-368x276.png);
+}
+
+.categories-red6 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/12/open-source-code-gq-368x276.png);
+}
+
+.categories-white7 {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(152, 154, 163, 0.52),
+      rgb(7, 1, 5)
+    ),
+    url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/12/toyota-suv-new-ev-car-future-electric-japan-368x276.jpg);
+}
+
+.categories-red7 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/10/hummer-ev-future-car-love-hate-next-gmc-gm-ultium.jpg);
+}
+
+.categories-white8 {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(152, 154, 163, 0.52),
+      rgb(7, 1, 5)
+    ),
+    url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/amplified-368x276.jpg);
+}
+
+.categories-red8 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/bloodlines.jpg);
+}
+
+/* Styling for latest deals section for mobile */
+.latest-deals {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.latest-deal-top {
+  display: flex;
+  justify-content: space-between;
+}
+
+.latest-deal-top a {
+  color: #f00;
+}
+
+.latest-deal-top a:hover {
+  color: #000;
+}
+
+.latest-deals-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 5px;
+}
+
+.latest-deal-item {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  width: 100%;
+}
+
+.item-pic {
+  width: 30%;
+  margin-right: 2%;
+}
+
+.item-desc {
+  width: 68%;
+}
+
+.item-image {
+  max-width: 100%;
+  display: block;
+}
+
+.latest-deal-item h4 {
+  font-size: 1.3rem;
+  font-weight: bold;
+}
+
+.red-hover:hover {
+  color: #f00;
+}
+
+.grey-text {
+  color: rgb(163, 157, 157);
+  font-size: 1.2rem;
+}
+
+.former-price {
+  text-decoration: line-through;
+  font-size: 0.9rem;
+  margin-right: 0.3rem;
+}
+
+.man-icon {
+  display: none;
+}
+
+/* TABLET */
+@media only screen and (min-width: 481px) {
```

Comment:
Hi guys, we are supposed to implement a mobile-first approach which is to have only 2 media queries which are for a tablet (768px) and desktop(1024px). There should be no media query for mobile device :computer: